pluggy/callers.py

```diff
@@ -97,6 +97,10 @@ def execute(self):
                 excinfo = sys.exc_info()
         finally:
             outcome = _Result(results, excinfo)
+            if firstresult:  # first result hooks return a single value
```

Comment:
i believe it would be more clear to construct the result differently in the firstresult case

```python
if firstresult:
   outcome = _Result(results[0] if results else None, excinfo)
else:
  outcome = _Result(results, excinfo)

Reply:
thanks ronny that's way cleaner 👍